chef/cookbooks/rabbitmq/recipes/default.rb

```diff
@@ -53,16 +53,17 @@
   rabbitmq_plugins = "#{RbConfig::CONFIG["libdir"]}/rabbitmq/bin/rabbitmq-plugins"
 end
 
-service "rabbitmq-server" do
-  supports :restart => true, :start => true, :stop => true
-  action [ :enable, :start ]
-  provider Chef::Provider::CrowbarPacemakerService if node[:rabbitmq][:ha][:enabled]
-end
-
 bash "enabling rabbit management" do
   environment "HOME" => "/root/"
-  code "#{rabbitmq_plugins} enable rabbitmq_management > /dev/null"
+  code "#{rabbitmq_plugins} --offline enable rabbitmq_management > /dev/null"
   not_if "#{rabbitmq_plugins} list -E | grep rabbitmq_management -q", :environment => {"HOME" => "/root/"}
   notifies :restart, "service[rabbitmq-server]"
 end
 
+service "rabbitmq-server" do
+  supports :restart => true, :start => true, :stop => true, :status => true
```

Comment:
yeah, we're using the systemd provider (which seems to have status enabled by default) and on sle11 we use the red Hat family. so this change is a noop for SUSE. I'd still like to keep it though.